spec/unit/provider/oneview_storage_pool_spec.rb

```diff
@@ -19,33 +19,54 @@
 provider_class = Puppet::Type.type(:oneview_storage_pool).provider(:c7000)
 api_version = login[:api_version] || 200
 resource_type = OneviewSDK.resource_named(:StoragePool, api_version, :C7000)
+system_type = OneviewSDK.resource_named(:StorageSystem, api_version, :C7000)
 
 describe provider_class, unit: true do
   include_context 'shared context'
+  context 'given the minimum parameters' do
+    let(:resource) do
+      Puppet::Type.type(:oneview_storage_pool).new(
+        name: 'Storage Pool',
+        ensure: 'present',
+        data:
+            {
+              'name' => '172.18.8.11, PDU 1',
```

Comment:
please remove the internal IP addresses.

Reply:
Done